test/EndToEnd/tests/BuildIntegratedTest.ps1

```diff
@@ -111,6 +111,9 @@ function Test-BuildIntegratedUninstallNonExistantPackage {
 }
 
 function Test-BuildIntegratedLockFileIsCreatedOnBuild {
+    [SkipTest('https://github.com/NuGet/Home/issues/12189')]
```

Comment:
I addressed this test failure in https://github.com/NuGet/Home/issues/12104 issue. My understanding is that this was an issue in 17.4, hence we ran PR builds on `17.3` for a week to unblock PR builds. I think Apex/E2E tests are failing on official builds due to https://developercommunity.visualstudio.com/t/Visual-Studio-freezes-and-needs-to-be-ki/10182087?

Reply:
Ok. I replaced with `PackCommand_PackConsoleAppWithRID_NupkgValid` test, there're more than 200 tests fail just 1-2 times in official pipeline doesn't fail in private pipeline. I found about 20 tests fails in both pipelines.